boule/_triaxialellipsoid.py

```diff
@@ -325,6 +341,27 @@ def meridional_flattening(self):
         """
         return (self.semimajor_axis - self.semiminor_axis) / self.semimajor_axis
 
+    def __str__(self):
+        s = self.name + " - " + self.long_name + "\n"
+        s += "Triaxial ellipsoid:\n"
+        s += f"  Semimajor axis: {self.semimajor_axis} m\n"
+        s += f"  Semimedium axis: {self.semimedium_axis} m\n"
+        s += f"  Semiminor axis: {self.semiminor_axis} m\n"
+        s += f"  Semiminor axis longitude: {self.semimajor_axis_longitude}\n"
+        s += f"  GM: {self.geocentric_grav_const} m³/s²\n"
+        s += f"  Angular velocity: {self.angular_velocity} rad/s"
```

Comment:
And similarly for the TriaxialEllipsoid:
```suggestion
        n_spaces = 24
        s = self.name + " - " + self.long_name + "\n"
        s += "Triaxial ellipsoid:\n"
        s += f"  • {'Semimajor axis':>{n_spaces}}: {self.semimajor_axis} m\n"
        s += f"  • {'Semimedium axis':>{n_spaces}}: {self.semimedium_axis} m\n"
        s += f"  • {'Semiminor axis':>{n_spaces}}: {self.semiminor_axis} m\n"
        s += (
            f"  • {'Semimajor axis longitude':>{n_spaces}}: "
            f"{self.semimajor_axis_longitude} °\n"
        )
        s += f"  • {'GM':>{n_spaces}}: {self.geocentric_grav_const} m³/s²\n"
        s += f"  • {'Angular velocity':>{n_spaces}}: {self.angular_velocity} rad/s"
```